bin/env_variables_all.txt

```diff
@@ -139,4 +139,5 @@ USER_PROFILE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_definition,ccd_admin
 XUI_LAUNCH_DARKLY_CLIENT_ID=1
 DB_USE_SSL=false
 ES_ENABLED_DOCKER=false
-IDAM_KEY_CCD_PS=AAAAAAAAAAAAAAAA
\ No newline at end of file
+IDAM_KEY_TS_TRANSLATION_SERVICE=AAAAAAAAAAAAAAAC
```

Comment:
This document was until recently in alphabetical order: please move new entry at least move to IDAM_KEY area.

Reply:
Can we keep the `IDAM_KEY_TS_TRANSLATION_SERVICE` value matching between here and hmcts/ts-translation-service#6.

Reply:
done